android/app/src/main/java/app/organicmaps/util/Speed.java

```diff
@@ -0,0 +1,39 @@
+package app.organicmaps.util;
+
+import android.content.Context;
+import android.util.Pair;
+
+import java.util.Locale;
+
+import app.organicmaps.Framework;
+import app.organicmaps.R;
+
+public class Speed
+{
+  public static double MpsToKmph(double mps) { return mps * 3.6; }
+  public static double MpsToMiph(double mps) { return mps * 2.236936; }
+
+  public static Pair<String, String> format(double speedInMetersPerSecond, int units,
+                                            Context context)
+  {
+    double speedValue;
+    String unitsString;
+
+    if (units == Framework.UNITS_IMPERIAL)
+    {
+      speedValue = MpsToMiph(speedInMetersPerSecond);
+      unitsString = context.getString(R.string.miles_per_hour);
```

Comment:
What if these strings will be loaded once (avoid calling context.getString on each call)?